lib/hca/military_information.rb

```diff
@@ -1,146 +0,0 @@
-# frozen_string_literal: true
-
-require 'va_profile/military_personnel/service'
-
-module HCA
-  class MilitaryInformation
-    PREFILL_METHODS = %w[
-      hca_last_service_branch
-      last_entry_date
-      last_discharge_date
-      discharge_type
-      post_nov111998_combat
-      sw_asia_combat
```

Comment:
I double checked and all of these exist in https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/va_profile/prefill/military_information.rb, which is what we want 🙂